windows-server-container-tools/Install-ContainerHost/Install-ContainerHost.ps1

```diff
@@ -1168,12 +1168,12 @@ mkdir %ProgramData%\docker
 
 :run
 if exist %certs%\server-cert.pem (if exist %ProgramData%\docker\tag.txt (goto :secure))
- 
+
 docker daemon -D -b "$global:SwitchName"
 goto :eof
- 
+
 :secure
-docker daemon -D -b "$global:SwitchName" -H 0.0.0.0:2376 --tlsverify --tlscacert=%certs%\ca.pem --tlscert=%certs%\server-cert.pem --tlskey=%certs%\server-key.pem
+docker daemon -D -b "$global:SwitchName" -H npipe:////./pipe/docker_engine -H 0.0.0.0:2376 --tlsverify --tlscacert=%certs%\ca.pem --tlscert=%certs%\server-cert.pem --tlskey=%certs%\server-key.pem
```

Comment:
Actually, thought about this some more. I think (not verified) that TLS will work over npipe, in which case the TCP listener shouldn't be required. Need someone like @taylorb-microsoft to verify if that is the case.